project/plugins.sbt

```diff
@@ -11,3 +11,5 @@ addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.5.0")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.2")
 
 addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.4.0")
+
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0") // "2.4.0" is just sbt plugin version
```

Comment:
Isn't it kinda obvious? ;D

Reply:
Oh ok, maybe change to `// "2.4.0" is just sbt plugin version and NOT scalafmt version itself` coz I guess that's what you mean

Reply:
scalafmt suggested this, because there is no connection between this version and the scalafmt version :) which for me is't obvious :D

Reply:
but thats the same case as with dotty so I will remove it.

Reply:
Pls change to what I suggested above, it's still worth leaving this comment IMO, just make it more clear

Reply:
Coz sbt plugins are notoriously inconsistent wrt. whether X sbt plugin vA.B.C provides X in version vA.B.C or not

Reply:
🆗